docs/howtos/runtime_sessions.rst

```diff
@@ -0,0 +1,70 @@
+Use Experiments with Runtime sessions
+=====================================
+
+Problem
+-------
+
+You want to run experiments in a `Runtime session
+<https://qiskit.org/ecosystem/ibm-runtime/sessions.html>`_ so that jobs can run in close temporal proximity.
+
+Solution
+--------
+
+There are two pathways currently supported:
+
+1. Use the :class:`~qiskit_ibm_provider.IBMBackend` object in ``qiskit-ibm-provider``, which supports sessions.
+
+.. jupyter-input::
+
+    from qiskit_ibm_provider import IBMProvider
+    from qiskit_experiments.library.tomography import ProcessTomography
+    from qiskit import QuantumCircuit
+
+    provider = IBMProvider()
+    backend = provider.get_backend("ibm_nairobi")
+    qc = QuantumCircuit(1)
+    qc.x(0)
+
+    with backend.open_session() as session:
+        tomography = ProcessTomography(qc)
+        tomography_result = tomography.run(backend)
+        tomography_result.block_for_results()
+        session.cancel()
```

Comment:
Since `session.close()` was just added in https://github.com/Qiskit/qiskit-ibm-provider/pull/762, we can also wait for the next provider release to update this line and merge. The difference seems to be `cancel()` will cancel all pending jobs while `close()` will prevent new jobs from being added but won't cancel pending ones. Since we're blocking for results here both are fine, but in general the user probably wants to use `close()` in a block like this.

Reply:
I would either wait or add a comment above this line like "Calling cancel because close is not available for `qiskit-ibm-provider<=0.7.2`. It is safe to call cancel since `block_for_results()` ensures there are no outstanding jobs still running that would be canceled." -- That's mainly to avoid someone leaving out the `block_for_results()` not realizing its importance.